src/main/resources/templates/slurm/command/sinfo

```diff
@@ -0,0 +1,5 @@
+sinfo
+[# th:if="${partitionName != null}"]
```

Comment:
I believe `${not #strings.isEmpty(partitionName)}` better here

Reply:
fixed